test-app/package.json

```diff
@@ -15,35 +15,50 @@
     "start": "ember serve",
     "test": "npm-run-all lint test:*",
     "test:ember": "ember test",
-    "test:ember-compatibility": "ember try:each"
+    "test:ember-compatibility": "pnpm qunit scenarios.js"
```

Comment:
there is no `ember-try` setup, hence made it so that `pnpm run test` can be used from test-app folder